MIGRATIONS.released.md

```diff
@@ -1,36 +1,9 @@
-# Migration Guide
+# Migration Guide (Released)
 All migrations of webKnossos are documented in this file.
+See `MIGRATIONS.unreleased.md` for the changes which are not yet part of an official release.
 
 This project adheres to [Calendar Versioning](http://calver.org/) `0Y.0M.MICRO`.
-User-facing changes are documented in the [changelog](CHANGELOG.md).
-
-## Unreleased
-- The optional `defaultOrganization` attribute from the `features` block in `application.conf` is not used anymore and can be removed. [#4559](https://github.com/scalableminds/webknossos/pull/4559)
-
-### Postgres Evolutions:
-
-## [20.04.0](https://github.com/scalableminds/webknossos/releases/tag/20.04.0) - 2020-03-23
```

Comment:
From here on it should be rather in the `released` file, I guess.

Reply:
Oh man, I thought I'd doublechecked everything properly and then this obvious mistake... Fixed it now. Thank you :)